src/styles/variables.css

```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css?family=Archivo:400,500,600,700&display=swap');
+
+$font-family: 'Archivo', sans-serif;
```

Comment:
We should add fallback to the font family other than `sans-serif`.
Something like this:
```css
font-family: "Archivo", Arial, "Helvetica Neue", Helvetica, sans-serif;
```